pkgs/stdenv/generic/make-derivation.nix

```diff
@@ -430,6 +430,14 @@ else let
         "/bin/sh"
       ];
       __propagatedImpureHostDeps = computedPropagatedImpureHostDeps ++ __propagatedImpureHostDeps;
+    } // lib.optionalAttrs (stdenv.targetPlatform.useAndroidPrebuilt) {
```

Comment:
This line has me wondering "what about non-prebuilt" android toolchains?

Reply:
I can change (and test) that to be android in general if wanted, though it would take me a bit to test it (and make sure stuff works).

Reply:
I guess there are few enough users that this isn't warranted. But if you could add a comment

> might work with android as well, but only tested with androidPrebuilt

That would probably help the next person coming along.